compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -823,6 +905,73 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
     post_trans->a(node);
     return true;
   }
+
+  bool visit(luci::CircleSub *node)
+  {
+    luci::CircleNode *pred_node = nullptr;
+    luci::CircleConst *subtract = nullptr;
+
+    if (is_NCHW_with_const_x(node, pred_node, subtract))
+    {
+      // case of subtract - pred_node
+      auto pre_trans = create_pre_transpose(node);
+      pre_trans->a(pred_node);
+
+      auto nhwc_const = create_NHWC_from_NCHW(subtract);
+      if (nhwc_const == nullptr)
+        return false;
+
+      node->x(nhwc_const);
+      node->y(pre_trans);
+    }
+    else if (is_NCHW_with_const_y(node, pred_node, subtract))
+    {
+      // case of pred_node - subtract
+      auto pre_trans = create_pre_transpose(node);
+      pre_trans->a(pred_node);
+
+      auto nhwc_const = create_NHWC_from_NCHW(subtract);
+      if (nhwc_const == nullptr)
+        return false;
+
+      node->x(pre_trans);
+      node->y(nhwc_const);
+    }
+    else if (subtract == nullptr)
+    {
+      // Both inputs are not constant.
```

Comment:
Is this true? It seems that both `is_NCHW_with_const_x` and `is_NCHW_with_const_y` can return false even when one of x/y is const.

Reply:
Hum... it was suspicious to me too... I'll revise this in another way...